src/client/pages/PlaceList/PlaceList.jsx

```diff
@@ -80,7 +81,24 @@ class PlaceList extends React.Component {
       .then(function (response) {
         instance.setState({ places: response.data });
       });
-    Promise.all([gettingTrip, gettingTripFriends, gettingPlaces])
+    const gettingUser = UserAPI.getUserData(this.routeChange, localStorage.getItem("userId"))
+      .then(function (response) {
+        instance.setState({
+          userData : response.data,
+        });
+      })
+      .then(() => {
+        let {username, name, profile_photo} = this.state.userData[0];
+        this.setState({
+          name: name,
+          username: username,
+          profilePictureLink: profile_photo ? profile_photo : '../../../../assets/common/user-icon.png',
```

Comment:
can u use the Ikono Mini icon instead of a picture? then remove this image from the assets? so that assets will be smaller